test/test_nix_handler.vader

```diff
@@ -0,0 +1,26 @@
+Execute(The nix handler should parse nix-instantiate error messages correctly):
```

Comment:
You'll need to make sure there's an `After` block here which resets the global linters list, or it could mess up the rest of the test suite run. I might just add it in.